Assets/Src/Scripts/SeedCalc/GameManager.cs

```diff
@@ -14,23 +14,89 @@
 
 using System;
 using UnityEngine;
+using UnityEngine.UI;
 using AgileMvvm;
 
 namespace SeedCalc {
   public class GameManager : MonoBehaviour {
-    private Calculator _calculator = null;
-
-    // The UI component of the calculator screen must be associated to this field in Unity.
     public CalculatorScreen Screen;
     public CuttingBoard CuttingBoard;
+    public GameObject SettingsDialog;
+    public GameObject AboutDialog;
+    public Sprite ActiveButtonSprite;
+    public Sprite InactiveButtonSprite;
+
+    private Calculator _calculator = null;
+    private Button _chineseButton = null;
+    private Button _englishButton = null;
+    private Button _soundSwitchButton = null;
+    private GameObject _soundOnText = null;
+    private GameObject _soundOffText = null;
 
     public void OnClickButton(string input) {
       if (_calculator.AcceptingInput) {
-        GetComponent<AudioSource>().Play();
+        PlayClickSound();
         _calculator.OnInput(input);
       }
     }
 
+    public void OnOpenAbout() {
+      PlayClickSound();
+      AboutDialog.SetActive(true);
+      LocalizationUtils.UpdateLocalizedAssets(AboutDialog.gameObject, true);
+    }
+
+    public void OnCloseAbout() {
+      PlayClickSound();
+      AboutDialog.SetActive(false);
+    }
+
+    public void OnOpenSettings() {
+      PlayClickSound();
+      SettingsDialog.SetActive(true);
+      LocalizationUtils.UpdateLocalizedAssets(SettingsDialog.gameObject, true);
+      string langCode = LocalizationUtils.GetCurrentLocale();
+      if (langCode == LocalizationUtils.ChineseLangCode) {
+        SetButtonState(_chineseButton, true);
```

Comment:
I tried code of this PR, but didn't see any difference of Chinese button and English button when its state is true or false. Have this feature be implemented in this PR?

Reply:
I cannot repro this.